test/test_files/sbol3_genbank_conversion/test_extra_properties_with_references.gb

```diff
@@ -1,15 +1,15 @@
-LOCUS       AF165912                5485 bp    DNA     linear   PLN 29-JUL-1999
+LOCUS       AF165912                5485 bp    DNA     circular PLN 29-JUL-1999
 DEFINITION  Arabidopsis thaliana CTP:phosphocholine cytidylyltransferase (CCT)
             gene, complete cds.
 ACCESSION   AF165912
 VERSION     AF165912.1  GI:5640000
-KEYWORDS    .
+KEYWORDS    test_keyword1; test_keyword2.
 SOURCE      Arabidopsis thaliana (thale cress)
   ORGANISM  Arabidopsis thaliana
-            Eukaryota; Viridiplantae; Streptophyta; Embryophyta; Tracheophyta;
-            Spermatophyta; Magnoliophyta; eudicotyledons; Gunneridae;
-            Pentapetalae; rosids; malvids; Brassicales; Brassicaceae;
-            Camelineae; Arabidopsis.
+            Arabidopsis; Brassicaceae; Brassicales; Camelineae; Embryophyta;
```

Comment:
This has ended up sorted, which is incorrect. It needs to stay in the order of the taxonomy hierarchy.

Reply:
Oh, I wasn't aware of this. Initially, I stored the taxonomy list as is in SBOL properties, but it led to a different ordering in the GenBank output for each conversion. Due to this, I tried storing and retrieving the taxonomy list in a sorted order always to maintain stability.

But if the order needs to be as it is originally, I can't figure why the order of lists gets changed when storing and when retrieving them from SBOL properties.

Reply:
When SBOL is read from an RDF file there is no ordering to the attributes. RDF does not guarantee any sort order. The object properties are more like sets than ordered lists. Round tripping SBOL data can result in different orders of property lists. 
If order _must_ be preserved, then ordering must be built on top of RDF/SBOL by somehow assigning an order to the elements in a property. There is no predefined way to do this.

Reply:
I can see two ways of doing this:
1. Kludge by doing this via a string rather than the list.
2. Make the Taxa an object as well, and give it an order field (similar to what we do for Location ordering).

The second is the better option, but will be a little more work up front.

Reply:
In the current state, the taxa are still listed in the wrong order.

Reply:
My bad, I have updated the test files in latest commit